Source/WebCore/Headers.cmake

```diff
@@ -603,6 +603,7 @@ set(WebCore_PRIVATE_FRAMEWORK_HEADERS
     Modules/url-pattern/URLPatternInit.h
     Modules/url-pattern/URLPatternOptions.h
     Modules/url-pattern/URLPatternResult.h
+    Modules/url-pattern/URLPatternTokenizer.h
```

Comment:
I doubt this new line is needed, or any of the `url-pattern/` ones above. This is only needed to export headers from WebCore/, so they can be used from WebKit/. None of `url-pattern/` is used from WebKit at the moment.